app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -55,6 +55,6 @@ def self.ems_type
   end
 
   def self.description
-    @description ||= "IBM Cloud".freeze
+    @description ||= "IBM Power Systems Virtual Servers".freeze
```

Comment:
This is a little bit confusing since it is different from our namespace `IbmCloud::PowerVirtualServers`. I think we need to match the branding exactly, and adding 'Systems' to the namespace would make it even longer.

Reply:
Yeah the description doesn't have to match the namespace exactly (branding tends to change more often than the backend :wink:) so I think this is fine

Reply:
Can you also put a PR in to https://github.com/ManageIQ/manageiq/blob/master/spec/models/ext_management_system_spec.rb#L69 to match?

Reply:
https://github.com/ManageIQ/manageiq/pull/20499